webrtc.html

```diff
@@ -5541,6 +5542,13 @@ <h2>Attributes</h2>
               <p>On getting, the attribute MUST return the value of the
               <a>[[\SenderRtcpTransport]]</a> slot.</p>
             </dd>
+            <dt><dfn><code>mstId</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
+            <dd>
+              <p>MediaStreamTrack id that appears in the msid attribute of the
```

Comment:
This means the second part of the MSID line.
Why don't you say that this is the msid-appdata token from draft-ietf-mmusic-msid, as carried in the SDP that is shown in the currentLocalDescription for this media section?
That makes it unambiguous.

(I also dislike the name "mstId", because it's an acronym - if you want to call it mediaStreamTrackId, I'll be happier - even though it's sometimes a bit misleading.)

Reply:
It makes sense, so I am updating it.
Thanks for the feedback ;).

Reply:
Done, but I did not mention anything about currentLocalDescription because IMO this attribute should have a value independently of the negotiation state [1].

Refs
[1] https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-currentlocaldescription